scripts/src/osd/sdl.lua

```diff
@@ -318,6 +318,11 @@ if BASE_TARGETOS=="unix" then
 			local str = backtick(sdlconfigcmd() .. " --libs")
 			addlibfromstring(str)
 			addoptionsfromstring(str)
+			if _OPTIONS["SDL_INSTALL_ROOT"]~=nil then
+				local str_cflags = backtick(sdlconfigcmd() .. " --cflags")
+				addincluderootfromstring(str_cflags, "/SDL2")
+				addcppoptionsfromstring(str_cflags)
+			end
```

Comment:
Should this only be done when `SDL_INSTALL_ROOT` is set?  In theory, the SDL installation automatically picked up by pkgconfig could theoretically require additional compile options – we’re just somewhat lucky that it doesn’t most of the time.  The only annoyance is that it will likely result in system include paths being added when SDL is installed in the default location.

Reply:
I have rebased the PR now. I will open another PR to fix parsing of SDL build flags once this is in master.